### PR TITLE
Add Config to Skip Cpp Codegen, Enable in FBCode

### PR DESCRIPTION
**Pull request by eellison**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #97198
* __->__ #97204

cc @soumith @voznesenskym @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @peterbell10 @desertfire

Differential Revision: [D44353662](https://our.internmc.facebook.com/intern/diff/D44353662)

### CODE REVIEW

**Review by ngimel — APPROVED** [review]

**Review by bertmaher — APPROVED** [review]
Nice!! Thank you!

**Review by mikekgfb — APPROVED** [review]
So cool!   Thank you!

**Review by cpuhrsch — APPROVED** [review]

**Comment by eellison** [comment]
@eellison has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D44353662).

**Comment by eellison** [comment]
@eellison has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D44353662).

**Comment by eellison** [comment]
@eellison has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D44353662).

**Comment by eellison** [comment]
@eellison has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D44353662).

**Comment by eellison** [comment]
@eellison has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D44353662).

**Comment by eellison** [comment]
@eellison has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D44353662).

**Comment by eellison** [comment]
@eellison has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D44353662).

File: torch/fx/passes/fake_tensor_prop.py [hunk 1]
```diff
@@ -29,10 +31,29 @@ def __init__(self, module: torch.fx.GraphModule, mode: Optional[FakeTensorMode]
...
+
+        def extract_val(obj):
+            if isinstance(obj, FakeTensor):
+                return snapshot_fake(obj)
+            elif isinstance(obj, torch.Tensor):
+                return snapshot_fake(self._mode.from_tensor(obj))
+            elif isinstance(obj, py_sym_types):
+                return obj
+            else:
+                return None
```
Comment (titaiwangms) [inline torch/fx/passes/fake_tensor_prop.py:43]: I am experimenting that using symbolic mode of fx.graph with FakeTensorProp to get shapes, and further use it to validate torch/onnx op level results inside a dynamic model. Before this PR, the obj (int) could be propagated to fx.node (ex: sym_size). Is here intentionally blocking the usage? If so, could you suggest an idiomatic way to do it?

**Pull request closed without merging**